cmd/uhc/get/cmd.go

```diff
@@ -72,11 +72,36 @@ func init() {
 
 func run(cmd *cobra.Command, argv []string) {
 	// Check that there is exactly one command line parameter:
-	if len(argv) != 1 {
-		fmt.Fprintf(os.Stderr, "Expected exactly one argument\n")
+	if len(argv) < 1 || len(argv) > 2 {
+		cmd.Usage()
+		fmt.Fprintf(os.Stderr, cmd.Usage().Error())
 		os.Exit(1)
 	}
-	path := argv[0]
+	preParsePath := argv[0]
+	path := preParsePath
+
+	switch preParsePath{
+
+	// list resources
```

Comment:
I try to keep the style of comments consistent in the project, starting them with upper case and ending with punctuation. Can you do that as well? I mean `// List resources:`, for example.

Reply:
done